tests/test_core_configure.py

```diff
@@ -126,7 +124,94 @@ def test_verify_spec_succeeds(capsys):
         mark:
         """
     spec = yaml.load(spec_yaml)
-    core_configure.verify_spec(spec)
+    errors = core_configure.verify_schema(spec)
+    assert len(errors) == 0
+
+
+def test_verify_spec_fails_multiple_roles_own_schema(capsys):
+    spec_yaml = """
+    jfinance:
+        owns:
+            schemas:
+                - finance_documents
+    jfauxnance:
+        owns:
+            schemas:
+                - finance_documents
+    """
+    spec = yaml.load(spec_yaml)
+    errors = core_configure.check_for_multi_schema_owners(spec)
+    expected = core_configure.MULTIPLE_SCHEMA_OWNER_ERR_MSG.format('finance_documents', 'jfinance, jfauxnance')
+    assert expected in errors
```

Comment:
We can be more stringent in our test here: `assert len(errors) == 1 and expected == errors[0]` or `assert [expected] == errors`. I'd just want to be sure there multiple errors aren't showing up and we aren't aware. Plus, if we can be more assertive why not be? The same holds for a couple other tests.